lib-es5/uploader.js

```diff
@@ -219,6 +219,29 @@ exports.create_zip = function create_zip(callback) {
   return exports.create_archive(callback, options, "zip");
 };
 
+exports.create_slideshow = function create_slideshow(options, callback) {
+  options.resource_type = "video";
+  return call_api("create_slideshow", callback, options, function () {
+    // Generate a transformation from the manifest_transformation key, which should be a valid transformation
+    var manifest_transformation = utils.generate_transformation_string(extend({}, options.manifest_transformation));
+
+    // Try to use all the options to generate a transformation (Example: options.width and options.height)
+    var transformation = utils.generate_transformation_string(extend({}, options));
```

Comment:
Since this is a new API, I would prefer it to behave according to spec, and check transformation only under `transformation` key, and without "best" effort that we do in other (legacy) apis.

Reply:
Agreed, fixed!

Reply:
Agreed, fixed!